num_enum_derive/src/lib.rs

```diff
@@ -94,11 +94,15 @@ struct VariantAlternativesAttribute {
 impl Parse for VariantAlternativesAttribute {
     fn parse(input: ParseStream) -> Result<Self> {
         let content;
+        let keyword = input.parse()?;
+        let _eq_token = input.parse()?;
+        let _bracket_token = syn::bracketed!(content in input);
+        let expressions = content.parse_terminated(Expr::parse)?;
         Ok(Self {
-            keyword: input.parse()?,
-            _eq_token: input.parse()?,
-            _bracket_token: syn::bracketed!(content in input),
-            expressions: content.parse_terminated(Expr::parse)?,
+            keyword,
+            _eq_token,
+            _bracket_token,
+            expressions,
```

Comment:
This is more verbose for no real good reason, imho; I guess it's just another instance of `clippy` being quite opinionated 🤷

Reply:
I buy that "initialisation order of struct fields isn't _super_ obvious" is a motivation, I don't feel particularly strongly either way but 🤷 I don't dislike it enough to work around it :)